src/pages/settings/Profile/LoginField.js

```diff
@@ -63,6 +64,15 @@ class LoginField extends Component {
         }
     }
 
+    /**
+    * Zero bottom margin for phone/email's label when it's unverified
+    * When phone/email is not verified the resend button increase the gap between label and text so the bottom margin is not required
```

Comment:
Sorry to be picky but I'd like to make the language slightly clearer here.

```suggestion
    * Bottom margin is not needed for phone/email label when unverified.
    * When phone/email is not verified, the resend button has a larger gap between the label and text so the bottom margin is not required.
```

Reply:
Also can you explain _why_ there is a larger gap when the phone/email is not verified? I don't think it's obvious but please let me know if I'm wrong :D

Reply:
This was pointed out [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1643727361370149?thread_ts=1643141719.423500&cid=C01GTK53T8Q). 
The space between the label and Text increases due to the presence of Resend Button. ([SS](https://files.slack.com/files-pri/T03SC9DTT-F031U2RABLY/screenshot_2022-02-01_at_8.27.54_pm.png))

Reply:
``` 
 * Bottom margin is not needed for phone/email label when unverified.
 * When phone/email is not verified, the resend button increases the gap between the label and text 
   so the bottom margin is not required.
  ```
  
  @Beamanator, let me know if the description looks good